src/resolvers/ResolverBuilder.ts

```diff
@@ -19,14 +18,14 @@ export class ResolverBuilder {
    * @param argumentContext context for generated arguments
    * @param knexContext name of knex object that was exposed
    */
-  constructor(argumentContext: string = 'resolve.args', knexContext: string = 'db') {
+  constructor(argumentContext: string = 'args', knexContext: string = 'db') {
```

Comment:
Since we control the entire implementation now we do not need these hacks. 
We can declare that args directly in the template.